src/indev/lv_indev.c

```diff
@@ -70,6 +70,7 @@ static void indev_proc_reset_query_handler(lv_indev_t * indev);
 static void indev_click_focus(lv_indev_t * indev);
 static void indev_gesture(lv_indev_t * indev);
 static bool indev_reset_check(lv_indev_t * indev);
+void indev_read(lv_indev_t * indev, lv_indev_data_t * data);
```

Comment:
If it's used only in this file, can we make it `static`? It would be nice to have new name for it which more different from `lv_indev_read()`. For example `read_core()` or so.

Reply:
Done. I have renamed `indev_read` to `indev_read_core`.